components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt

```diff
@@ -93,13 +93,12 @@ class FirefoxAccount(handle: FxaHandle, persistCallback: PersistCallback?) : Aut
      * This performs network requests, and should not be used on the main thread.
      *
      * @param scopes List of OAuth scopes for which the client wants access
-     * @param wantsKeys Fetch keys for end-to-end encryption of data from Mozilla-hosted services
      * @return String that resolves to the flow URL when complete
      */
-    fun beginOAuthFlow(scopes: Array<String>, wantsKeys: Boolean): String {
+    fun beginOAuthFlow(scopes: Array<String>): String {
```

Comment:
Is it possible to continue accepting (but ignore) a `wantsKeys` argument as a nod to backwards-compatibility, or do we have enough b/w-incompatible changes here already that it's not worth it?

Reply:
`android-components` wraps this call so I think we should just break the API.